docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+if [ -f tmp/pids/server.pid ]; then
+  rm tmp/pids/server.pid
+fi
```

Comment:
This could just be:

`rm -f tmp/pids/server.pid`

Reply:
This @jakerenzella for you suggestion. It's done.